cmd/gateway-main.go

```diff
@@ -209,6 +209,7 @@ func gatewayMain(ctx *cli.Context) {
 	endpointAddr := ctx.Args().Get(1)
 	// Third argument is the address flag
 	serverAddr := ctx.String("address")
+	fatalIf(CheckLocalServerAddr(serverAddr), "Invalid address ‘%s’ in command line argument.", serverAddr)
```

Comment:
@harshavardhana, I don't understand why the original submitter alone should fix this. Is there any specific reason for this policy?

Are we only fixing show stopper issues now? This fix is simple and it's easy to understand its impact. I don't see a reason to wait for another release. Further, the point about duplicate fix is weak for a one line fix. Close the PR if you still feel strongly.